CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 1.55.1 (2019-10-23)
```

Comment:
Might be inaccurate once it's released

Reply:
In US Midway it's still 23rd! ;-)

Reply:
> Wait for it...

![image](https://user-images.githubusercontent.com/1606004/67482457-102c1e00-f664-11e9-9067-0dabbab66699.png)

Reply:
I released it yesterday :)